superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -231,7 +310,7 @@ const ViewResultsModalTrigger = ({
               <Button
                 buttonStyle="primary"
                 buttonSize="small"
-                onClick={closeModal}
+                onClick={() => setShowModal(false)}
```

Comment:
is that change necessary? I think it's the same as `closeModal`

Reply:
thanks for catching! Yes, I removed the `closeModal` method in one of the iterations and forgot to put it back.